survey/4-tech.yml

```diff
@@ -313,5 +335,20 @@ questions:
     required: true
     choices:
       - Daily
+      - Occasionally
       - Tried them but not interested
       - Never
+
+  - label: "If you are using AI tools for Dev, How much improvement do you see in your productivity?"
+    required: true
+    choices:
+      - Not noticeable
+      - 2X - 3X
+      - 4X - 5X
+      - +5X
+
+  - label: "Do you think AI tools for Dev will replace developers in the future?"
+    required: true
+    choices:
+      - Yes
+      - No
```

Comment:
here is my suggestion. for tools i think we already mentioned them in the platforms and tools questions 

for others, I would suggest two question 
1. If you have the chance to work on developing an AI solution, did you use
* In-house model 
* Open source model 
* 3rd party model 
* Hybrid model ( combination of those options)

2. Did you start learning about AI 
* Yes.
* No.
* Considering it.

Reply:
There's some very good qsts here: https://retool.com/reports/state-of-ai-2023
Tbh I'd love to have a specific section just for AI with as many qsts as possible from the above 😍
Otherwise if you think it's too much, these are some of the most important ones imo:
- Company's level of AI adoption
- Motivations to pursue AI
- AI usecases in these companies
- Pain points around developing AI apps
- Most popular AI dev tools / models

Reply:
Love these 🙌 is there a way to have a dedicated AI section @yjose !?